src/components/dashboard/WhoContent.native.js

```diff
@@ -0,0 +1,203 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react'
+import { FlatList, PermissionsAndroid } from 'react-native'
+import { promisify } from 'es6-promisify'
+import contacts from 'react-native-contacts'
+import { map, memoize, orderBy, uniq } from 'lodash'
+import { isAndroid } from '../../lib/utils/platform'
+import { Section } from '../common'
+import InputText from '../common/form/InputText'
+import { withStyles } from '../../lib/styles'
+import normalize from '../../lib/utils/normalizeText'
+import { getDesignRelativeHeight } from '../../lib/utils/sizes'
+import userStorage from '../../lib/gundb/UserStorage'
+import FeedContactItem from './FeedContactItem'
+import ItemSeparator from './ItemSeparator'
+const Contacts = promisify(contacts.getAll)
+
+const WhoContent = ({ styles, setContact, error, text, value, next, state, showNext, setValue }) => {
+  const [contacts, setContacts] = useState([])
+  const [recentFeedItems, setRecentFeedItems] = useState([])
+  const [recentlyUsedList, setRecentlyUsedList] = useState([])
+  let initialList = useRef()
```

Comment:
could be const, as we mutating it's `.current` property not itself
but it's not critical

Reply:
you can edit code and commit inline such small changes

Reply:
@serdiukov-o-nordwhale